Dockerfile

```diff
@@ -1,23 +1,18 @@
-ARG BASE_IMAGE=docker.io/fedora:32
+FROM quay.io/ansible/python-builder:latest as builder
```

Comment:
The quay.io pages have no descriptions. Sorry for being so out of the loop here, but can I get the Containerfile/Dockerfile from which this is being built? This will be very important to us to maintain on our own team as a part of the pipelines for producing execution environments, and I know I have already seen a lot of fixes that impact functionality in the collection install process.

Reply:
Yah, I just started on publishing it, I need to better understand how the readme for that works.  https://github.com/ansible/python-builder-image is the repo for now, I do agree adding the right people is the way to go.

Right now it is setup to just deal with bindep / requirements files. The next step for me, would be to grow ansible-collections support. But I also think talking more about it is good.